source/part1/chapter-02/md/00-pandas-basics.md

```diff
@@ -220,18 +225,14 @@ data.shape
 ```
```

Comment:
Are the inline-comments in the code cells necessary 
`# Check the number of rows` 
`# Check dataframe shape (number of rows, number of columns)`

(could not leave this comment inside the code cell..)

Reply:
Let's indeed try to avoid comments in the code blocks. Only if having longer pieces of code, then we should comment, so that the reader understands what is going on.